pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -408,8 +408,11 @@ func (r *NetworkInfoReconciler) listNamespaceCRsNameIDSet(ctx context.Context) (
 	nsSet := sets.Set[string]{}
 	idSet := sets.Set[string]{}
 	for _, ns := range namespaces.Items {
-		nsSet.Insert(ns.Name)
-		idSet.Insert(string(ns.UID))
+		// Ignore the terminating Namespaces in the list results.
+		if ns.DeletionTimestamp.IsZero() {
```

Comment:
ns.ObjectMeta.DeletionTimestamp.IsZero()?

Reply:
They should take the same effect. Since only field ObjectMeta has property "DeletionTimeStamp", when we access Namespace.DeletionTimestamp, it actually accesses Namespace.ObjectMeta.DeletionTimestamp.

Reply:
Yes. that is the same.

Reply:
ok, got it.